data-model/fdo-profiles/0.1.0/handle-kernel/schema/handle-kernel.json

```diff
@@ -17,14 +16,13 @@
       "$comment": "idx = 2"
     },
     "digitalObjectType": {
-      "type": "string",
       "description": "PID to a description of the Type of digital object that defines the metadata, bit sequences (if any) and operations for the object",
-      "example": "https://hdl.handle.net/21.T11148/894b1e6cad57e921764e",
+      "const": "https://hdl.handle.net/21.T11148/532ce6796e2828dd2be6",
```

Comment:
Ok yeah, so we need to update these when we get the prod PID

Reply:
A bit of a chicken and the egg thing going on here. We have to finalize the schema to send it to epic and get a PID, but we can't finalize the schema without a PID from epic...I'll explain this in the email I send to them. hopefully they can mint PIDs first and send them to us. Otherwise, we'd have to leave this as an open string, which isn't ideal.

Reply:
Yeah sorry it was more a remark but of course